oh-my-bash.sh

```diff
@@ -30,56 +30,56 @@ fi
 
 # Load all of the config files in ~/.oh-my-bash/lib that end in .sh
 # TIP: Add files you don't want in git to .gitignore
-for config_file in $OSH/lib/*.sh; do
+for config_file in "$OSH"/lib/*.sh; do
   custom_config_file="${OSH_CUSTOM}/lib/${config_file:t}"
   [ -f "${custom_config_file}" ] && config_file=${custom_config_file}
-  source $config_file
+  source "$config_file"
 done
 
 
 is_plugin() {
   local base_dir=$1
   local name=$2
-  test -f $base_dir/plugins/$name/$name.plugin.sh \
-    || test -f $base_dir/plugins/$name/_$name
+  test -f "$base_dir/plugins/$name/$name.plugin.sh" \
+    || test -f "$base_dir/plugins/$name/_$name"
```

Comment:
For conditionals, isn't it better to use the special construct `[[ ... ]]` rather than quoting the arguments?